nipype/algorithms/events.py

```diff
@@ -0,0 +1,407 @@
+from __future__ import division
+from builtins import range
+
+from copy import deepcopy
+import os
+
+from nibabel import load
+import numpy as np
+from nipype.external.six import string_types
+from nipype.interfaces.base import (BaseInterface, TraitedSpec, InputMultiPath,
+                               traits, File, Bunch, BaseInterfaceInputSpec,
+                               isdefined, OutputMultiPath)
+from nipype import config, logging
+import re
+from glob import glob
+from os.path import basename
+import json
+iflogger = logging.getLogger('interface')
+
+# Eventually, move the following inside the interface, or wrap in an import check
+try:
+    import pandas as pd
+except ImportError:
```

Comment:
perhaps simply create a flag here: `no_pandas = True` and raise it in the `run_interface` function below